diff --git a/ports/boost/CONTROL b/ports/boost/CONTROL
index 964665b..f54dbfb 100644
--- a/ports/boost/CONTROL
+++ b/ports/boost/CONTROL
@@ -1,4 +1,4 @@
 Source: boost
 Version: 1.64-5
 Description: Peer-reviewed portable C++ source libraries
-Build-Depends: zlib, bzip2
+Build-Depends: zlib, bzip2, python2 [windows]
diff --git a/ports/boost/desktop/user-config.jam b/ports/boost/desktop/user-config.jam
index 8faba6d..3bd7502 100644
--- a/ports/boost/desktop/user-config.jam
+++ b/ports/boost/desktop/user-config.jam
@@ -5,3 +5,7 @@ using msvc : 14.0 : cl.exe :
 using msvc : 14.1 : cl.exe :
 <setup>echo
 ;
+
+using python : 2.7
+: @PYTHON2_DIR@
+;
\ No newline at end of file
diff --git a/ports/boost/portfile.cmake b/ports/boost/portfile.cmake
index 8db5442..52f6c17 100644
--- a/ports/boost/portfile.cmake
+++ b/ports/boost/portfile.cmake
@@ -79,10 +79,22 @@ set(B2_OPTIONS
     --hash
     -q
 
-    --without-python
+    # --without-python
     threading=multi
 )
 
+find_program(PYTHON2 NAMES python2 python PATHS C:/python27 E:/Python27 c:/Python27amd64 ENV PYTHON)
+if (PYTHON2 MATCHES "NOTFOUND")
+    list(APPEND B2_OPTIONS --without-python)
+else()
+    get_filename_component(PYTHON2_DIR ${PYTHON2} DIRECTORY)
+    set(PIP "${PYTHON2_DIR}/Scripts/pip")
+    vcpkg_execute_required_process(COMMAND ${PIP} install --user numpy
+        WORKING_DIRECTORY ${PYTHON2_DIR}
+        LOGNAME pip-${TARGET_TRIPLET}
+    )
+endif()
+
 if (VCPKG_CRT_LINKAGE STREQUAL dynamic)
     list(APPEND B2_OPTIONS runtime-link=shared)
 else()
