diff --git a/ports/opencv/portfile.cmake b/ports/opencv/portfile.cmake
index b6958ab..888b3d1 100644
--- a/ports/opencv/portfile.cmake
+++ b/ports/opencv/portfile.cmake
@@ -68,6 +68,24 @@ if(VCPKG_CMAKE_SYSTEM_NAME STREQUAL "WindowsStore")
   set(WITH_MSMF OFF)
 endif()
 
+set(CMAKE_FIND_LIBRARY_PREFIXES "")
+set(CMAKE_FIND_LIBRARY_SUFFIXES ".lib")
+
+find_package(PythonInterp 3.6)
+if (PYTHONINTERP_FOUND)
+  find_package(PythonLibs 3.6)
+  set(PYTHON3_OPTIONS -DBUILD_opencv_python3=ON
+                      -DPYTHON3_EXECUTABLE=${PYTHON_EXECUTABLE}
+                      -DPYTHON3_INCLUDE_DIR=${PYTHON_INCLUDE_DIR}
+                      -DPYTHON3_LIBRARY=${PYTHON_LIBRARY}
+                      -DPYTHON3_LIBRARY_DEBUG=${PYTHON_LIBRARY_DEBUG}
+                      -DPYTHON3_LIBRARIES=${PYTHON_LIBRARIES}
+                      -DPYTHON3_DEBUG_LIBRARIES=${PYTHON_DEBUG_LIBRARIES}
+                      -DPYTHON_DEFAULT_EXECUTABLE=${PYTHON_EXECUTABLE})
+else()
+  set(PYTHON3_OPTIONS -DBUILD_opencv_python3=OFF)
+endif()
+
 vcpkg_configure_cmake(
     SOURCE_PATH ${SOURCE_PATH}
     OPTIONS
@@ -90,9 +108,11 @@ vcpkg_configure_cmake(
         -DBUILD_ZLIB=OFF
         -DBUILD_opencv_apps=OFF
         -DBUILD_opencv_dnn=ON
+        -DBUILD_opencv_dnn_modern=OFF
         -DBUILD_opencv_flann=ON
         -DBUILD_opencv_python2=OFF
-        -DBUILD_opencv_python3=OFF
+        # -DBUILD_opencv_python3=ON
+        ${PYTHON3_OPTIONS}
         -DBUILD_opencv_sfm=${BUILD_opencv_sfm}
         # CMAKE
         -DCMAKE_DISABLE_FIND_PACKAGE_JNI=ON
@@ -123,6 +143,7 @@ vcpkg_configure_cmake(
         -DINSTALL_OTHER=OFF
 )
 
+# set(_VCPKG_CMAKE_GENERATOR "Visual Studio")
 vcpkg_install_cmake()
 
 file(COPY ${SOURCE_PATH}/LICENSE DESTINATION ${CURRENT_PACKAGES_DIR}/share/opencv)
