diff --git a/ports/opencv/portfile.cmake b/ports/opencv/portfile.cmake
index 78ed5fa..bafda49 100644
--- a/ports/opencv/portfile.cmake
+++ b/ports/opencv/portfile.cmake
@@ -236,6 +236,24 @@ endif()
 
 string(REPLACE ";" "\\\\\;" CMAKE_MODULE_PATH "${CMAKE_MODULE_PATH}")
 
+set(CMAKE_FIND_LIBRARY_PREFIXES "")
+set(CMAKE_FIND_LIBRARY_SUFFIXES ".lib")
+
+find_package(PythonInterp 3.6)
+if (PYTHONINTERP_FOUND)
+  find_package(PythonLibs 3.6)
+  set(PYTHON3_OPTIONS -DBUILD_opencv_python3=ON
+                      -DPYTHON3_EXECUTABLE=${PYTHON_EXECUTABLE}
+                      -DPYTHON3_INCLUDE_DIR=${PYTHON_INCLUDE_DIR}
+                      -DPYTHON3_LIBRARY=${PYTHON_LIBRARY}
+                      -DPYTHON3_LIBRARY_DEBUG=${PYTHON_LIBRARY_DEBUG}
+                      -DPYTHON3_LIBRARIES=${PYTHON_LIBRARIES}
+                      -DPYTHON3_DEBUG_LIBRARIES=${PYTHON_DEBUG_LIBRARIES}
+                      -DPYTHON_DEFAULT_EXECUTABLE=${PYTHON_EXECUTABLE})
+else()
+  set(PYTHON3_OPTIONS -DBUILD_opencv_python3=OFF)
+endif()
+
 vcpkg_configure_cmake(
     PREFER_NINJA
     SOURCE_PATH ${SOURCE_PATH}
@@ -263,11 +281,13 @@ vcpkg_configure_cmake(
         -DBUILD_opencv_apps=OFF
         -DBUILD_opencv_bgsegm=${BUILD_opencv_bgsegm}
         -DBUILD_opencv_dnn=${BUILD_opencv_dnn}
+        -DBUILD_opencv_dnn_modern=OFF
         -DBUILD_opencv_flann=${BUILD_opencv_flann}
         -DBUILD_opencv_line_descriptor=${BUILD_opencv_line_descriptor}
         -DBUILD_opencv_ovis=${BUILD_opencv_ovis}
         -DBUILD_opencv_python2=OFF
-        -DBUILD_opencv_python3=OFF
+        # -DBUILD_opencv_python3=OFF
+        ${PYTHON3_OPTIONS}
         -DBUILD_opencv_saliency=${BUILD_opencv_saliency}
         -DBUILD_opencv_sfm=${BUILD_opencv_sfm}
         # PROTOBUF
@@ -313,6 +333,7 @@ vcpkg_configure_cmake(
         -DINSTALL_OTHER=OFF
 )
 
+# set(_VCPKG_CMAKE_GENERATOR "Visual Studio")
 vcpkg_install_cmake()
 
 file(COPY ${SOURCE_PATH}/LICENSE DESTINATION ${CURRENT_PACKAGES_DIR}/share/opencv)
