diff --git a/triplets/x64-windows-static.cmake b/triplets/x64-windows-static.cmake
index 75f9b95..0dc68ff 100644
--- a/triplets/x64-windows-static.cmake
+++ b/triplets/x64-windows-static.cmake
@@ -1,3 +1,4 @@
 set(VCPKG_TARGET_ARCHITECTURE x64)
 set(VCPKG_CRT_LINKAGE static)
 set(VCPKG_LIBRARY_LINKAGE static)
+set(VCPKG_PLATFORM_TOOLSET v140)
diff --git a/triplets/x64-windows.cmake b/triplets/x64-windows.cmake
index d0be729..90aeb74 100644
--- a/triplets/x64-windows.cmake
+++ b/triplets/x64-windows.cmake
@@ -1,4 +1,4 @@
 set(VCPKG_TARGET_ARCHITECTURE x64)
 set(VCPKG_CRT_LINKAGE dynamic)
 set(VCPKG_LIBRARY_LINKAGE dynamic)
-
+set(VCPKG_PLATFORM_TOOLSET v140)
diff --git a/triplets/x86-windows-static.cmake b/triplets/x86-windows-static.cmake
index c855f0c..4990a60 100644
--- a/triplets/x86-windows-static.cmake
+++ b/triplets/x86-windows-static.cmake
@@ -1,3 +1,4 @@
 set(VCPKG_TARGET_ARCHITECTURE x86)
 set(VCPKG_CRT_LINKAGE static)
-set(VCPKG_LIBRARY_LINKAGE static)
\ No newline at end of file
+set(VCPKG_LIBRARY_LINKAGE static)
+set(VCPKG_PLATFORM_TOOLSET v140)
diff --git a/triplets/x86-windows.cmake b/triplets/x86-windows.cmake
index 40ccbe1..1f0ca58 100644
--- a/triplets/x86-windows.cmake
+++ b/triplets/x86-windows.cmake
@@ -1,3 +1,4 @@
 set(VCPKG_TARGET_ARCHITECTURE x86)
 set(VCPKG_CRT_LINKAGE dynamic)
 set(VCPKG_LIBRARY_LINKAGE dynamic)
+set(VCPKG_PLATFORM_TOOLSET v140)
