diff --git a/scripts/buildsystems/vcpkg.cmake b/scripts/buildsystems/vcpkg.cmake
index 71cf4e9..df17f74 100644
--- a/scripts/buildsystems/vcpkg.cmake
+++ b/scripts/buildsystems/vcpkg.cmake
@@ -229,3 +229,9 @@ if(NOT _CMAKE_IN_TRY_COMPILE)
         "set(_VCPKG_ROOT_DIR \"${_root_dir}\" CACHE STRING \"\")\n"
         )
 endif()
+
+set(FindwxWidgetsCmakeDir ${_VCPKG_INSTALLED_DIR}/${VCPKG_TARGET_TRIPLET}/share/wxwidgets)
+if (EXISTS ${FindwxWidgetsCmakeDir}/FindwxWidgets.cmake)
+    set(CMAKE_MODULE_PATH ${FindwxWidgetsCmakeDir} ${CMAKE_MODULE_PATH})
+endif()
+unset(FindwxWidgetsCmakeDir)
