diff --git a/ports/wxwidgets/portfile.cmake b/ports/wxwidgets/portfile.cmake
index 1568826..c76ae4b 100644
--- a/ports/wxwidgets/portfile.cmake
+++ b/ports/wxwidgets/portfile.cmake
@@ -63,7 +63,7 @@ set(NMAKE_OPTIONS_DBG
     CFG=${TARGET_TRIPLET}-dbg
 )
 
-file(REMOVE_RECURSE ${SOURCE_PATH}/lib/${LIB_SUB_PATH})
+# file(REMOVE_RECURSE ${SOURCE_PATH}/lib/${LIB_SUB_PATH})
 
 ################
 # Release build
@@ -94,11 +94,29 @@ file(INSTALL ${SOURCE_PATH}/include
     DESTINATION ${CURRENT_PACKAGES_DIR})
 file(INSTALL ${SOURCE_PATH}/lib/${LIB_SUB_PATH}-rel/mswu/wx/setup.h DESTINATION ${CURRENT_PACKAGES_DIR}/include/wx)
 file(INSTALL ${SOURCE_PATH}/lib/${LIB_SUB_PATH}-rel/mswu/wx/msw/rcdefs.h DESTINATION ${CURRENT_PACKAGES_DIR}/include/wx/msw)
-file(INSTALL ${SOURCE_PATH}/lib/${LIB_SUB_PATH}-rel/
-    DESTINATION ${CURRENT_PACKAGES_DIR}/lib FILES_MATCHING PATTERN "*.lib" PATTERN "*.pdb")
-file(INSTALL ${SOURCE_PATH}/lib/${LIB_SUB_PATH}-dbg/
-    DESTINATION ${CURRENT_PACKAGES_DIR}/debug/lib FILES_MATCHING PATTERN "*.lib" PATTERN "*.pdb")
-file(REMOVE_RECURSE ${CURRENT_PACKAGES_DIR}/lib/mswu ${CURRENT_PACKAGES_DIR}/debug/lib/mswud)
+
+if (VCPKG_LIBRARY_LINKAGE STREQUAL "static")
+    file(INSTALL ${SOURCE_PATH}/lib/${LIB_SUB_PATH}-rel/
+        DESTINATION ${CURRENT_PACKAGES_DIR}/lib FILES_MATCHING PATTERN "*.lib" PATTERN "*.pdb")
+    file(INSTALL ${SOURCE_PATH}/lib/${LIB_SUB_PATH}-dbg/
+        DESTINATION ${CURRENT_PACKAGES_DIR}/debug/lib FILES_MATCHING PATTERN "*.lib" PATTERN "*.pdb")
+    file(REMOVE_RECURSE ${CURRENT_PACKAGES_DIR}/lib/mswu ${CURRENT_PACKAGES_DIR}/debug/lib/mswud)
+else()
+    file(INSTALL ${SOURCE_PATH}/lib/${LIB_SUB_PATH}-rel/
+        DESTINATION ${CURRENT_PACKAGES_DIR}/bin FILES_MATCHING PATTERN "*.dll" PATTERN "*.pdb")
+    file(INSTALL ${SOURCE_PATH}/lib/${LIB_SUB_PATH}-rel/
+        DESTINATION ${CURRENT_PACKAGES_DIR}/lib FILES_MATCHING PATTERN "*.lib")
+
+    file(INSTALL ${SOURCE_PATH}/lib/${LIB_SUB_PATH}-dbg/
+        DESTINATION ${CURRENT_PACKAGES_DIR}/debug/bin FILES_MATCHING PATTERN "*.dll" PATTERN "*.pdb")
+    file(INSTALL ${SOURCE_PATH}/lib/${LIB_SUB_PATH}-dbg/
+        DESTINATION ${CURRENT_PACKAGES_DIR}/debug/lib FILES_MATCHING PATTERN "*.lib")
+
+    file(REMOVE_RECURSE ${CURRENT_PACKAGES_DIR}/bin/mswu ${CURRENT_PACKAGES_DIR}/debug/bin/mswud)
+    file(REMOVE_RECURSE ${CURRENT_PACKAGES_DIR}/lib/mswu ${CURRENT_PACKAGES_DIR}/debug/lib/mswud)
+endif()
+
+file(INSTALL ${CMAKE_CURRENT_LIST_DIR}/FindwxWidgets.cmake DESTINATION ${CURRENT_PACKAGES_DIR}/share/wxwidgets)
     
 # Handle copyright
 file(COPY ${SOURCE_PATH}/docs/licence.txt DESTINATION ${CURRENT_PACKAGES_DIR}/share/wxwidgets)
